### ISSUE TITLE
 PublishWithi18nAction ignores validation errors

### ISSUE THREAD

**Issue by DennisRosen** [LiamMartens/sanity-plugin-intl-input#46]
Not sure if I am using this plugin incorrectly, but the custom publish action from this plugin does not seem to handle document validation errors at all, resulting in invalid documents being published.

**Comment by nickgriffiths** [comment]
I am also experiencing this issue.

**Comment by LiamMartens** [comment]
Fixed in 5.2.0

**Comment by daniel-akerman-eqtpartners** [comment]
@LiamMartens It's now reacting on warnings as well. Normally warnings will not prevent you from publishing. I suggest changing to something like

``` 
  const hasValidationErrors = validationStatus.markers.some(
    marker => marker.level === 'error'
  )
```
